test/Sandpit/Exscientia/Align/test_merge.py

```diff
@@ -0,0 +1,88 @@
+from typing import Dict, Optional
+
+import BioSimSpace.Sandpit.Exscientia as BSS
+import pytest
+from BioSimSpace.Sandpit.Exscientia.Align._merge import _removeDummies
+from BioSimSpace.Sandpit.Exscientia.Process import Gromacs
+from BioSimSpace.Sandpit.Exscientia.Protocol import Production
+from BioSimSpace.Sandpit.Exscientia.Units.Temperature import kelvin
+from BioSimSpace.Sandpit.Exscientia.Units.Time import femtosecond
+from BioSimSpace.Sandpit.Exscientia._SireWrappers import Molecule
+
+
+def get_energy(
+    mol: Molecule,
+    coord_mol: Optional[Molecule] = None,
+    coord_mol_mapping: Optional[Dict[int, int]] = None,
+) -> float:
+    if coord_mol is not None:
+        c = mol._sire_object.cursor()
+        c_coord = coord_mol._sire_object.cursor()
+        coordinates = c_coord["coordinates"].toVector()
+        if coord_mol_mapping is not None:
+            sorted_keys = sorted(coord_mol_mapping)
+            coordinates = [coordinates[coord_mol_mapping[k]] for k in sorted_keys]
+        for atom, coord in zip(c.atoms(), coordinates):
+            atom["coordinates"] = coord
+        mol = Molecule(c.commit())
+
+    system = mol.toSystem()
+
+    protocol = Production(
+        timestep=0.1 * femtosecond,
+        runtime=0.1 * femtosecond,
+        temperature=0 * kelvin,
+    )
+
+    proc = Gromacs(system, protocol)
+    proc.start()
+    proc.wait()
+
+    res = proc.getPotentialEnergy().value()
+
+    return res
+
+
+@pytest.fixture
+def benzene():
```

Comment:
Would it be better if these move to conftest?

Reply:
I agree - ideally all fixtures should be in conftest so they don't get accidentally duplicated.